Clojure/Clojure.sublime-settings

```diff
@@ -0,0 +1,7 @@
+{
+  "word_separators": "/\\()[]{}\"'`,:;@#^~.",
+  "smart_indent": false,
+  "detect_indentation": false,
+  "tab_size": 1,
```

Comment:
This seems funny to me. Why would the tab size be hardcoded to 1 for all Clojure files?

Reply:
Most Clojure code follows the same indentation convention, which sometimes involves 1-space indents. The [Lispindent](https://packagecontrol.io/packages/lispindent) package automates this style, but when refactoring, you often need to shift a block by 1 space, and it's needlessly painful if the indentation is not set to 1.

Example from the test file: [[1]](https://github.com/Mitranim/Packages/blob/c035c0a72b273069eba8dafb0962c8f0434e8fd9/Clojure/tests/syntax_test_clojure.clj#L2050-L2051).

Reply:
The most common reason for indent 1 is when writing nested data structures, such as Hiccup-style markup:

```clj
(def page []
  [:div.class0
   [:div.class1
    [:div.class2
     ...]]])
```

Reply:
If most code is formatted that way, the automatic detection would be able to catch this, wouldn't it? I'd be very vary of overriding indentation settings for all users, unless the format basically requires you to use it (kind of like YAML).